extension-base/src/main/java/com/azure/autorest/extension/base/model/codemodel/Scheme.java

```diff
@@ -0,0 +1,80 @@
+package com.azure.autorest.extension.base.model.codemodel;
```

Comment:
copyright

just config it in IDE

I will see if I got time to write a script to add them to existing classes and enable the checkstyle rule...

Reply:
checkstyle rule merged

Reply:
Thanks, will update